pkgs/development/web/nodejs/v18.nix

```diff
@@ -16,6 +16,8 @@ let
     buildPackages = buildPackages // { stdenv = ensureCompatibleCC buildPackages; };
     python = python311;
   };
+
+  python312Patches = callPackage ./python312-v18-patches.nix { };
```

Comment:
We're currently pinning Python 3.11 already, can we drop this for now?

Reply:
Some of the Python 3.12 commits were backported and released upstream (see https://github.com/nodejs/node/blob/v18.20.3/doc/changelogs/CHANGELOG_V18.md#commits-3), I’m not really sure whether we should be using Python 3.11 instead of fully backporting other changes required for Python 3.12 🤔

Reply:
We don't want any more rebuilds for this staging-next, in the future we'll want to switch to 3.12.